server/index.js

```diff
@@ -0,0 +1,452 @@
+'use strict';
+
+var util = require('util');
+
+var envvar = require('envvar');
+var express = require('express');
+var bodyParser = require('body-parser');
+var moment = require('moment');
+var plaid = require('plaid');
+
+var APP_PORT = envvar.number('APP_PORT', 8000);
+var PLAID_CLIENT_ID = envvar.string('PLAID_CLIENT_ID');
+var PLAID_SECRET = envvar.string('PLAID_SECRET');
```

Comment:
We may  want to make this `PLAID_SANDBOX_SECRET` to make it easier to know what value to use here since the sample app and env below are set to sandbox or at least leave a comment that the secret here must match the environment below